instructor-notes/08_pull-requests-basics.md

```diff
@@ -35,29 +35,47 @@ In the most recent session, participants will have created two branches:
 
 * Checkout the `add-renv` branch locally.
 * We will make a small change to `download-fastq.sh` to skip downloading the `fastq` files if they already exist.
-    * Check if files exist using the `-e` flag as outlined in [`solutions/fileexists_download-fastq.sh`](solutions/fileexists_download-fastq.sh).
-    This sets us up to have a merge conflict with the `add-fastp` branch in the next session.
+  * Check if files exist using the `-e` flag as outlined in [`solutions/fileexists_download-fastq.sh`](solutions/fileexists_download-fastq.sh).
+  This sets us up to have a merge conflict with the `add-fastp` branch in the next session.
 * Commit the change to the `add-renv` branch.
 * Push the changes to the remote `add-renv` branch.
 * Instruct participants to navigate to their remote repositories in the browser to see their new commit pushed.
   * Point out the banner that says changes were recently pushed to the `add-renv` branch.
 
+### Protecting `main`
+
+* Navigate to the settings of the repository, where you will demonstrate how to protect the `main` branch.
+* In the left sidebar, click on "Branches".
+* Under "Branch protection rules", click on "Add branch ruleset".
```

Comment:
Do we want to have them start by clickin on `Rules` instead? The reason is in part because that is what I highlighted in #65, but also because that is where they will see the rules once created (they don't show up in the branches tab, for no good reason I can see).

Reply:
Maybe add a point at the end showing them where the rules can be found for editing?

Reply:
Added the point at the end in https://github.com/AlexsLemonade/2025-treehouse-training/pull/64/commits/aeafa67cf3e4dc2c3b5cc90f5cea929caff9bf70